### ISSUE TITLE
Create User Manual and Tutorial

### ISSUE THREAD

**Issue by petya-kangalova** [hotosm/fAIr#132]
**Is your feature request related to a problem? Please describe.**
Create clear step by step guide for users with screenshots

**Describe the solution you'd like**
@kshitijrajsharma @omranlm  our suggestion is to host the tutorial on Github and then connect to this page :
https://fair-dev.hotosm.org/learn  Double checking the info there now is not hosted anywhere on Github?

@neelimagoogly  will be happy to work on this but would need support, screenshots and also any live demo. She can work on this in late July as other docs are prioritized now. Please add any useful resources below.

**Comment by kshitijrajsharma** [comment]
Yes ! @petya-kangalova Sure ! I  love the idea , Lets go for it, Let me know if I can  help with tutorials / demo

**Comment by haruharu1017** [comment]
Hi please assign this to me

**Comment by KimFarida** [comment]
@kshitijrajsharma I would also like to take on this issue, if it has not already been assigned please

**Comment by kshitijrajsharma** [comment]
Sure go ahead both !
